gdal/gcore/gdal.h

```diff
@@ -133,6 +133,7 @@ typedef enum
     /*! Cubic B-Spline Approximation (4x4 kernel) */    GRIORA_CubicSpline = 3,
     /*! Lanczos windowed sinc interpolation (6x6 kernel) */ GRIORA_Lanczos = 4,
     /*! Average */                                      GRIORA_Average = 5,
+    /*! Average */                                      GRIORA_AverageQuadratic = 13,
```

Comment:
comment to edit, and value to set to 14, so we have a match between the GDALResampleAlg enum  and this one. Comment a bit below "NOTE : values 8 to 12..." should be edited to be "NOTE : values 8 to 13 are reserved for max,min,med,Q1,Q3,sum"